app/assets/stylesheets/active_admin/components/_comments.scss

```
@@ -4,7 +4,7 @@
   .active_admin_comment {
     clear: both;
```

This change should fix the issue more reliably across different sized comments:
```suggestion
    @include clearfix;
```